src/DB_Command.php

```
@@ -2393,4 +2434,319 @@ protected function apply_sql_mode_compat_init_command( &$mysql_args, $assoc_args
+
+		if ( $is_row_modifying_query ) {
+			// phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+			$affected_rows = $wpdb->query( $query );
+			if ( false === $affected_rows ) {
+				// phpcs:ignore WordPress.WP.AlternativeFunctions.strip_tags_strip_tags
+				WP_CLI::error( 'Query failed: ' . strip_tags( $wpdb->last_error ) );
+			}
+			WP_CLI::success( "Query succeeded. Rows affected: {$affected_rows}" );
+		} else {
+			// phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+			$results = $wpdb->get_results( $query, ARRAY_A );
```

PHPStan (2.2.5)

[error] Constant ARRAY_A not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)